ec/src/scalar_mul/variable_base/mod.rs

```diff
@@ -44,6 +44,14 @@ pub trait VariableBaseMSM: ScalarMul {
     fn msm_bigint(
         bases: &[Self::MulBase],
         bigints: &[<Self::ScalarField as PrimeField>::BigInt],
+    ) -> Self {
+        Self::unhooked_msm_bigint(bases, bigints)
+    }
+
+    /// /// Raw optimized implementation of multi-scalar multiplication, only for usage by curve models.
+    fn unhooked_msm_bigint(
```

Comment:
I don't think we need to introduce an extra wrapper method. That adds an unnecesary layer of abstraction.